modules/slide-box/slide-box-bb-2-2-compatibility.php

```diff
@@ -493,7 +493,6 @@
 						'back_background_color'  => array(
 							'type'        => 'color',
 							'label'       => __( 'Background Color', 'uabb' ),
-							'default'     => '',
```

Comment:
any specific reason why default is removed?

Reply:
yes, the default key is already provided on line 499